pippo-core/src/main/java/ro/pippo/core/HttpConstants.java

```diff
@@ -28,11 +28,6 @@
         public static final String HEAD = "HEAD";
         public static final String PUT = "PUT";
         public static final String PATCH = "PATCH";
-        /*
```

Comment:
This block is commented because for now we don't want to activate these methods. I would like to keep this comment.